dashboard/modules/reporter/reporter_head.py

```diff
@@ -76,6 +76,10 @@ async def get_cluster_status(self, req):
         These fields are both read from the GCS, it's expected that the
         autoscaler writes them there.
         """
+        # TODO(rickyx): We should be able to get the cluster status from the
+        # autoscaler directly with V2. And we should be able to return structured data
+        # rather than a string.
```

Comment:
Does this already pre-process pg resources btw? (that's prerequisite before returning the structured data via this API)

Reply:
wdym by `pre-process` pg resources?

Reply:
ah I meant the pg resources need to be parsed and structured (e.g., CPU_group_id_... things)

Reply:
yeah, it's parsed (by the legacy code). So nothing is changed here as of now.